src/encoded/schemas/gene.json

```diff
@@ -122,6 +204,56 @@
             },
             "permission": "import_items"
         },
+        "locations": {
+            "title": "Gene location coordinates",
+            "description": "Genomic locations for different assembles of this gene.",
+            "type": "array",
+            "default": [],
+            "uniqueItems": true,
+            "items": {
+                "title": "Gene Location Coordinate",
```

Comment:
I think to be consistent it should be "Gene location coordinate" and not "Gene Location Coordinate"